packages/server-ct/src/routes-ct.ts

```diff
@@ -58,12 +68,14 @@ module.exports = ({ app, config, project, onError }) => {
   // during routing just log them out to
   // the console and send 500 status
   // and report to raygun (in production)
-  app.use((err, req, res) => {
+  const errorHandlingMiddleware: ErrorRequestHandler = (err, req, res, next) => {
```

Comment:
Unused next parameter? Is there a reason typescript needs you to pass this through?
```suggestion
  const errorHandlingMiddleware: ErrorRequestHandler = (err, req, res) => {
```